lib/pengine/unpack.c

```diff
@@ -451,12 +451,35 @@ pcmk_node_t *
 pe_create_node(const char *id, const char *uname, const char *type,
                const char *score, pcmk_scheduler_t *scheduler)
 {
+    enum pcmk__node_variant variant = pcmk__node_variant_cluster;
     pcmk_node_t *new_node = NULL;
 
     if (pcmk_find_node(scheduler, uname) != NULL) {
         pcmk__config_warn("More than one node entry has name '%s'", uname);
     }
 
+    if (pcmk__str_eq(type, PCMK_VALUE_MEMBER,
+                     pcmk__str_null_matches|pcmk__str_casei)) {
+        variant = pcmk__node_variant_cluster;
+
+    } else if (pcmk__str_eq(type, PCMK_VALUE_REMOTE, pcmk__str_casei)) {
+        variant = pcmk__node_variant_remote;
+        pcmk__set_scheduler_flags(scheduler, pcmk__sched_have_remote_nodes);
```

Comment:
the flag should be set after the node is allocated in case we return on error

Reply:
whoops, thanks